general/update_fars/01_download_fars.py

```diff
@@ -0,0 +1,15 @@
+from urllib import request
+import os
+
+# place to save data locally
+data_folder = "/mnt/c/Users/tputta/OneDrive - Toole Design/Desktop/SSPF/FARS"
+
+start_yr = 2015
+end_yr = 2021
+
+for yr in range(start_yr, end_yr+1):
+    print(f"Downloading {yr} FARS data")
+    if not os.path.exists(f"{data_folder}/{yr}"):
+        os.mkdir(f"{data_folder}/{yr}")
+    request.urlretrieve(f"https://static.nhtsa.gov/nhtsa/downloads/FARS/{yr}/National/FARS{yr}NationalCSV.zip", f"{data_folder}/{yr}/FARS{yr}NationalCSV.zip")
+    yr += 1
```

Comment:
Why are we incrementing `yr` here? Won't it just go to the next value once it goes back to the top of the loop?

Reply:
I started with while loop and this is a vestige of that that I forgot to remove. It does not really harm anything in this case, but I will remove it